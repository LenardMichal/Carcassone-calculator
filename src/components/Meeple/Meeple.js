import React from 'react';


const Meeple = props => {
  return <div>
  </div>
}

export default Meeple;