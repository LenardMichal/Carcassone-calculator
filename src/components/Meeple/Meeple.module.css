
.main{
  width: 100%;
  display: flex;
  height: auto;
  color: white;
  align-items: center; 
  justify-content: center;
  flex-wrap: wrap;
  user-select: none;
  cursor: pointer;
}
              
.active::after{
  content: 'A';
  width: 100%;
  position: absolute;
  margin-bottom: 8vh;
}