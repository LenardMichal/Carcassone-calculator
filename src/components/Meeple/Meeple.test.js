import React from 'react';
import renderer from 'react-test-renderer';
import Meeple from './Meeple';

describe('test of Meeple component', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Meeple />).toJSON();

    expect(tree).toMatchSnapshot();
  })
})