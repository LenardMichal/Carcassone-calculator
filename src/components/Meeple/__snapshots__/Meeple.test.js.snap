// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test of Meeple component should have valid snapshot 1`] = `
<div>
  Adolf K.
</div>
`;
